python-checks/src/main/java/org/sonar/python/checks/cdk/UnencryptedSqsQueueCheck.java

```diff
@@ -23,38 +23,25 @@
 import org.sonar.plugins.python.api.SubscriptionContext;
 import org.sonar.plugins.python.api.tree.CallExpression;
 
-import static org.sonar.python.checks.cdk.CdkPredicate.isFqn;
+import static org.sonar.python.checks.cdk.CdkPredicate.isFalse;
 import static org.sonar.python.checks.cdk.CdkPredicate.isNone;
 import static org.sonar.python.checks.cdk.CdkUtils.getArgument;
 
 @Rule(key = "S6330")
 public class UnencryptedSqsQueueCheck extends AbstractCdkResourceCheck {
-  private static final String UNENCRYPTED_MESSAGE = "Setting \"encryption\" to \"QueueEncryption.UNENCRYPTED\" disables SQS queues encryption. Make sure it is safe here.";
-  private static final String NONE_MESSAGE = "Setting \"encryption\" to \"None\" disables SQS queues encryption. Make sure it is safe here.";
-  private static final String OMITTING_MESSAGE = "Omitting \"encryption\" disables SQS queues encryption. Make sure it is safe here.";
+  private static final String SQS_MANAGED_DISABLED_MESSAGE = "Setting \"sqs_managed_sse_enabled\" to \"false\" disables SQS queues encryption. Make sure it is safe here.";
   private static final String CFN_NONE_MESSAGE = "Setting \"kms_master_key_id\" to \"None\" disables SQS queues encryption. Make sure it is safe here.";
-  private static final String CFN_OMITTING_MESSAGE = "Omitting \"kms_master_key_id\" disables SQS queues encryption. Make sure it is safe here.";
 
   @Override
   protected void registerFqnConsumer() {
-    checkFqn("aws_cdk.aws_sqs.Queue", this::checkQueue);
     checkFqn("aws_cdk.aws_sqs.CfnQueue", this::checkCfnQueue);
   }
 
-  protected void checkQueue(SubscriptionContext ctx, CallExpression resourceConstructor) {
-    getArgument(ctx, resourceConstructor, "encryption").ifPresentOrElse(
-      flow -> {
-        flow.addIssueIf(isFqn("aws_cdk.aws_sqs.QueueEncryption.UNENCRYPTED"), UNENCRYPTED_MESSAGE);
-        flow.addIssueIf(isNone(), NONE_MESSAGE);
-      },
-      () -> ctx.addIssue(resourceConstructor.callee(), OMITTING_MESSAGE)
-    );
-  }
-
   protected void checkCfnQueue(SubscriptionContext ctx, CallExpression resourceConstructor) {
-    getArgument(ctx, resourceConstructor, "kms_master_key_id").ifPresentOrElse(
-      flow -> flow.addIssueIf(isNone(), CFN_NONE_MESSAGE),
-      () -> ctx.addIssue(resourceConstructor.callee(), CFN_OMITTING_MESSAGE)
-    );
+    getArgument(ctx, resourceConstructor, "sqs_managed_sse_enabled")
+        .ifPresent(flow -> flow.addIssueIf(isFalse(), SQS_MANAGED_DISABLED_MESSAGE));
+
+    getArgument(ctx, resourceConstructor, "kms_master_key_id")
+      .ifPresent(flow -> flow.addIssueIf(isNone(), CFN_NONE_MESSAGE));
```

Comment:
This case should not be checked. If `kms_master_key_id` is`None`, SQS SSE will be used instead (like if `kms_master_key_id` was not present) and thus the queue will be encrypted and the code is compliant.

Reply:
Thank you Sebastien for the review I have made the change!